src/test/java/com/glovoapp/planningpoker/e2e/pages/SessionPage.java

```diff
@@ -0,0 +1,233 @@
+    private static final By TICKET_SELECTOR = By.id("ticket-name");
+    private static final By ERROR_MESSAGE_SELECTOR = By.id("error-message");
+    private static final By AUTO_RELOAD_MESSAGE_SELECTOR = By.id("auto-reload-message");
+
+    private static final By CLEAR_BUTTON_SELECTOR = By.id("clear-all");
+    private static final By SHOW_VOTES_SELECTOR = By.id("show-votes");
+    private static final By SHARE_SESSION_LINK_SELECTOR = By.id("copy-session-link-to-clipboard");
+
+    private static final By YOUR_VOTE_SELECTOR = By.id("current-vote");
+    private static final By VOTE_BUTTONS_SELECTOR = By.id("vote-buttons");
+    private static final By MEDIAN_VOTES_SELECTOR = By.id("median-votes-text");
+    private static final By AVERAGE_VOTES_SELECTOR = By.id("average-votes-text");
+    private static final By VOTES_TABLE_SELECTOR = By.id("votes-table");
+    private static final By DANCE_SELECTOR = By.id("glopi-dance");
+
+    public void waitForError() {
+        final WebDriverWait wait = new WebDriverWait(driver, 2);
+        wait.until(visibilityOfElementLocated(ERROR_MESSAGE_SELECTOR));
+    }
+
+    private WebElement danceSection() {
+        return driver.findElement(DANCE_SELECTOR);
+    }
+
+    public void isHeReallyDancing() {
+        danceSection().findElement(By.tagName("img")).isDisplayed();
+    }
+
+    private List<WebElement> getVotingButtons() {
+        final WebElement votingButtons = driver.findElement(VOTE_BUTTONS_SELECTOR);
+        return votingButtons.findElements(By.tagName("button"));
+    }
+
+    public Optional<WebElement> getVotingButton(final String voteValue) {
+        return getVotingButtons().stream()
+                .filter(element -> voteValue.equals(element.getText()))
+                .findFirst();
+    }
+
+    private WebElement getYourNameIDField() {
+        return driver.findElement(YOUR_NAME_SELECTOR);
+    }
+
+    public void clickNameField() {
+        driver.findElement(YOUR_NAME_SELECTOR).click();
+    }
+
+    public void clearNameField() {
+        driver.findElement(YOUR_NAME_SELECTOR).clear();
+    }
+
+    private WebElement getErrorMessageIdField() {
+        return driver.findElement(ERROR_MESSAGE_SELECTOR);
+    }
+
+    public String getErrorMessage() {
+        return getErrorMessageIdField().getText();
+    }
+
+    public void enterName(final String userName) {
+        getYourNameIDField().sendKeys(userName);
+    }
+
+    private WebElement getTicketIDField() {
+        return driver.findElement(TICKET_SELECTOR);
+    }
+
+    public void enterTicket(final String ticketName) {
+        getTicketIDField().sendKeys(ticketName);
+    }
+
+    public void compareTicket(final String expectedText) { assertEquals(expectedText, getTicketIDField().getAttribute("value"));;}
+
+    private WebElement getSpectatorIdField() {
+        return driver.findElement(SPECTATOR_SELECTOR);
+    }
+
+    public void specClick() {
```

Comment:
Please don't use abbreviations for method or variable names:
```suggestion
    public void clickSpectatorToggle() {
```